src-docs/src/views/icon/icon_example.js

```diff
@@ -213,46 +194,50 @@ export const IconExample = {
           graph color).
         </p>
       ),
-      snippet: iconColorsSnippet,
       demo: <IconColors />,
     },
+    {
+      wrapText: false,
+      text: (
+        <>
+          <EuiText>
+            <p>
+              Two-tone icons, like our app style icons, will behave similarly to
+              normal glyphs when provided a specific color by applying the color to
+              <strong>all</strong> the shapes within.
```

Comment:
We're missing a space here:
<img width="256" alt="Screenshot 2021-04-13 at 10 24 24" src="https://user-images.githubusercontent.com/2750668/114530065-80f13c80-9c42-11eb-96f2-7ab8f25b6eb0.png">

```suggestion
              {' '}<strong>all</strong> the shapes within.
```